### PR TITLE
CP 6.0 GA Release

### PR DESCRIPTION
**Pull request by JumaX**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Confluent Platform 6.0 GA release.

Fixes # (issue)

See release notes for further details: https://docs.confluent.io/current/release-notes/index.html

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

New molecule scenarios have been added for each piece of new functionality.  All scenarios have been run and completed successfully.

Upgrades across all major platforms have been performed and passed testing.

See HOWTOTEST.md for further details.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: roles/confluent.common/defaults/main.yml [hunk 1]
```diff
@@ -54,14 +54,9 @@ jolokia_jar_url: "http://search.maven.org/remotecontent?filepath=org/jolokia/jol
...
-
-### Full path on hosts to install the Confluent CLI
-confluent_cli_path: /usr/local/bin/confluent
-
 ### A path reference to a local archive file or URL. By default this is the URL from Confluent's repositories. In an ansible-pull deployment this could be set to a local file such as "~/.ansible/pull/{{inventory_hostname}}/{{confluent_archive_file_name}}".
-confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}-{{confluent_archive_scala_version}}.tar.gz"
-
+# confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}-{{confluent_archive_scala_version}}.tar.gz"
+## Uncomment above line and change back for 6.0 release
+confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}.tar.gz"
```
Comment (domenicbove) [inline roles/confluent.common/defaults/main.yml:60]:
```suggestion
confluent_archive_file_source: "{{confluent_common_repository_baseurl}}/archive/{{confluent_repo_version}}/confluent-{{confluent_package_version}}.tar.gz"
```

remove commented code
also run `python doc.py` to update VARIABLES.md and commit that

File: roles/confluent.test/molecule/Dockerfile-ubuntu.j2 [hunk 2]
```diff
@@ -4,21 +4,44 @@ RUN mkdir -p /usr/share/man/man1 && \
...
+      confluent-kafka-connect-replicator=6.0.0-1 \
+      confluent-kafka-rest=6.0.0-1 \
+      confluent-ksqldb=6.0.0-1 \
+      confluent-schema-registry=6.0.0-1 \
+      confluent-security=6.0.0-1 \
+      confluent-control-center-fe=6.0.0-1 \
+      confluent-control-center=6.0.0-1 \
+      confluent-schema-registry=6.0.0-1
+
+# RUN wget -qO - https://s3-us-west-2.amazonaws.com/staging-confluent-packages-6.0.0/deb/6.0/archive.key | sudo apt-key add - && \
```
Comment (domenicbove) [inline roles/confluent.test/molecule/Dockerfile-ubuntu.j2:28]: remove commented out code

File: roles/confluent.test/molecule/Dockerfile-debian.j2 [hunk 3]
```diff
@@ -12,21 +12,43 @@ RUN apt-get update && \
...
+     confluent-kafka-connect-replicator=6.0.0-1 \
+     confluent-kafka-rest=6.0.0-1 \
+     confluent-ksqldb=6.0.0-1 \
+     confluent-schema-registry=6.0.0-1 \
+     confluent-security=6.0.0-1 \
+     confluent-control-center-fe=6.0.0-1 \
+     confluent-control-center=6.0.0-1 \
+     confluent-schema-registry=6.0.0-1
+
+# RUN wget -qO - https://s3-us-west-2.amazonaws.com/staging-confluent-packages-6.0.0/deb/6.0/archive.key | sudo apt-key add - && \
```
Comment (domenicbove) [inline roles/confluent.test/molecule/Dockerfile-debian.j2:35]: remove commented out code

**Review by domenicbove — APPROVED** [review]
Wohoo lgtm

**Review by andrewegel — APPROVED** [review]

**Pull request merged**